packages/ra-core/src/auth/useLogoutIfAccessDenied.spec.tsx

```diff
@@ -79,56 +76,67 @@ describe('useLogoutIfAccessDenied', () => {
     });
 
     it('should not logout if passed no error', async () => {
-        const { queryByText } = renderInRouter(
-            <AuthContext.Provider value={authProvider}>
-                <TestComponent />
-            </AuthContext.Provider>
-        );
+        render(<Route path="/" element={<TestComponent />} />, {
+            wrapper: TestWrapper,
+        });
 
         await waitFor(() => {
             expect(authProvider.logout).toHaveBeenCalledTimes(0);
             expect(notify).toHaveBeenCalledTimes(0);
-            expect(queryByText('logged in')).not.toBeNull();
+            expect(screen.queryByText('logged in')).not.toBeNull();
         });
     });
 
     it('should not log out if passed an error that does not make the authProvider throw', async () => {
-        const { queryByText } = renderInRouter(
-            <AuthContext.Provider value={authProvider}>
-                <TestComponent error={new Error()} />
-            </AuthContext.Provider>
+        render(
+            <Route path="/" element={<TestComponent error={new Error()} />} />,
+            {
```

Comment:
is prettier tired or did you intentionally make it less readable? ;)

Reply:
prettier